embrace-gradle-plugin/src/main/java/io/embrace/android/gradle/plugin/api/EmbraceBytecodeInstrumentation.kt

```diff
@@ -41,6 +41,16 @@ abstract class EmbraceBytecodeInstrumentation @Inject internal constructor(objec
      */
     val autoSdkInitializationEnabled: Property<Boolean> = objectFactory.property(Boolean::class.java)
 
+    /**
+     * Whether Embrace should automatically instrument application initialization timing start. Defaults to true.
+     */
+    val applicationInitTimeStartEnabled: Property<Boolean> = objectFactory.property(Boolean::class.java).convention(true)
+
+    /**
+     * Whether Embrace should automatically instrument application initialization timing end. Defaults to true.
+     */
+    val applicationInitTimeEndEnabled: Property<Boolean> = objectFactory.property(Boolean::class.java).convention(true)
```

Comment:
If these are true then presumably we want to set `end_startup_with_app_ready` automatically to true @bidetofevil?  https://embrace.io/docs/android/features/performance-instrumentation/#configuration